python/hsfs/storage_connector.py

```diff
@@ -587,6 +587,36 @@ def prepare_spark(self, path: Optional[str] = None):
     def _get_path(self, sub_path: str):
         return os.path.join(self.path, sub_path)
 
+    def read(
+        self,
+        query: str = None,
+        data_format: str = None,
+        options: dict = {},
+        path: str = "",
+    ):
+
+        """Reads a path into a dataframe using the storage connector.
+        # Arguments
+            query: Not relevant for ADLS connectors.
+            data_format: The file format of the files to be read, e.g. `csv`, `parquet`.
+            options: Any additional key/value options to be passed to the ADLS connector.
+            path: Path within the bucket to be read. For example, `Path/to/file` will read directly from the default container specified on connector. To read from another container, you can also specify full URI .e.g. 'abfss://[container-name]@[account_name].dfs.core.windows.net/[path]'.
+            If no path is specified default container path will be used from connector.
+
+        # Returns
+            `DataFrame`.
+        """
+        path = path.strip()
+        if not path.startswith("abfss://") or path.startswith("adl://"):
+            path = self._get_path(path)
+            print(
+                "Prepending default bucket specified on connector, final path: {}".format(
```

Comment:
`bucket` is not a term for adls containers.

Reply:
done